PProPanel/jsx/general.jsx

```diff
@@ -0,0 +1,6 @@
+if (typeof $ == 'undefined') {
```

Comment:
>@bbb999 is the $ object always injected?
While I am not @bbb999, I will say that `$` is a global object defined in the ExtendScript extensions to JavaScript. It is part of the environment. It _may_ be possible to **reassign** it without replacing it, but I've never seen an example of this in ExtendScript.

I would suggest that these files get changed such that the check that `$` exists be replaced with a comment along the lines of:

```javascript
// Extensions defined herein are grafted onto ExtendScript's global "$" object.
```